src/core/connectivity.jl

```diff
@@ -11,6 +11,10 @@ This connectivity type is encountered in `QPUs` such as the [`AnyonYukonQPU`](@r
 # Fields
 - `dimension         ::Int` -- Qubit count in this connectivity.
 - `excluded_positions::Vector{Int}` -- Optional: List of qubits on the connectivity which are disabled, and cannot be interacted with. Elements in Vector must be unique.
+- `excluded_connections::Vector{Tuple{Int, Int}}` -- Optional: List of couplers on the connectivity which are disabled, and cannot be interacted with. Elements in Vector must be unique.
```

Comment:
lets update the comment: List of connections between qubits which are disabled, over which 2-qubit gates cannot be performed.

Reply:
I'll change it to: "List of connections between qubits which are disabled and cannot perform 2-qubit gates."